src/Pickles/Pickles.ObjectModel/ObjectModel/Scenario.cs

```diff
@@ -44,6 +44,8 @@ public Scenario()
 
         public Feature Feature { get; set; }
 
+        public Location Location { get; set; }
+
         #endregion
```

Comment:
This is not strictly your code, but do me a favour and remove the `#region` from the class. Thanks!